lib/jsdom/living/helpers/form-controls.js

```diff
@@ -80,8 +80,45 @@ exports.normalizeToCRLF = string => {
     .replace(/^\n/, "\r\n");
 };
 
+// https://html.spec.whatwg.org/multipage/dom.html#interactive-content-2
+exports.isInteractiveContent = node => {
+  if (node.nodeType !== NODE_TYPE.ELEMENT_NODE) {
+    return false;
+  }
+
+  if (node.hasAttributeNS(null, "tabindex")) {
+    return true;
+  }
+  switch (node.tagName) {
```

Comment:
Stil nit: prefer localName, as the fundamental thing, to tagName, the derived thing.

More substantive issue: I think a HTML namespace check is needed for all of these.

I realize both of these follow the `isLabelable` implementation below. It would be _ideal_ to update both, and add tests (for non-HTML elements of those tag names) for both... but the tests at least are not required. I'd just rather us do the better-practice thing for new code, and preferably update the nearby old code at the same time.